### ISSUE TITLE
About the size of data used for training and evaluation

### ISSUE THREAD

**Issue by RICKand-MORTY** [DeepXuan/DuDoDp-MAR#8]
Hi! May I ask if you used the complete SynDeepLesion dataset for training and evaluation? Specifically, the 347.16G training set and the 8.13G test set? Thank you!
您好，请问在训练和评估时是使用了完整的SynDeepLesion数据集吗？即347.16G大小的训练集和8.13G的测试集
